File: INF.04-01-22.06-SG/konsola/konsola.py
import random
random.seed()

class Algorytm:

    def __init__(self, dlugosc, szukana):    
        self.tablica = [0] * ( dlugosc + 1 )
        self.dlugosc = dlugosc
        self.szukana = szukana

    def wczytaj(self):
        print('Twoja tablica: ')
        for i in range(self.dlugosc):
            self.tablica[i] = int(random.random()*100)
            print(self.tablica[i])
        self.tablica[self.dlugosc] = self.szukana

    def szukaj(self):
        for i in range(self.dlugosc):
            if (self.tablica[i] == self.szukana):
                return i;
        return self.dlugosc;



n = 51  # dlugosc tablicy

x = int(input('Podaj wartość którą chcesz wyszukać '))
a1 = Algorytm(n, x)

a1.wczytaj()

index = a1.szukaj()

if (index == n):
    print('Twoja liczba: nie została znaleziona w tym zbiorze')
else:
    print('Twoja liczba została znaleziona na miejscu: ' + str(index))

File: INF.04-01-22.06-SG/konsola/konsola.cs
using System;

class Algorytm
{
    private int[] tab;

    public Algorytm(int n)
    {
        tab = new int[n];
    }

    public void Wczytaj(int n)
    {
        Random random = new Random();
        Console.WriteLine("Twoja tablica:");
        for (int i = 0; i < n; i++)
        {
            tab[i] = random.Next(0, 101);
            Console.Write(tab[i] + ", ");
        }
        Console.WriteLine();
    }

    public int Szukaj(int x, int n)
    {
        for (int i = 0; i < n; i++)
        {
            if (tab[i] == x)
            {
                return i;
            }
        }
        return n;
    }
}

class Program
{
    static void Main()
    {
        const int n = 51; // Długość tablicy
        Algorytm a1 = new Algorytm(n);

        Console.WriteLine("Wpisz liczbe z zakresu od 1 do 100 ktora chcesz wyszukac:");
        int x = int.Parse(Console.ReadLine());

        a1.Wczytaj(n);
        int index = a1.Szukaj(x, n);

        if (index == n)
        {
            Console.WriteLine("\nTwoja liczba nie zostala znaleziona w tym zbiorze.");
        }
        else
        {
            Console.WriteLine($"\nTwoja liczba zostala znaleziona na miejscu: {index}");
        }
    }
}